api/envoy/extensions/transport_sockets/proxy_protocol/v3/upstream_proxy_protocol.proto

```diff
@@ -33,4 +48,8 @@ message ProxyProtocolUpstreamTransport {
   // If true, all the TLVs are encoded in the connection pool key.
   // [#not-implemented-hide:]
   bool tlv_as_pool_key = 4;
+
+  // Custom TLV metadata to be sent in the upstream PROXY protocol header.
+  // [#not-implemented-hide:]
```

Comment:
Why isn't this implemented?

Reply:
Addressed in https://github.com/envoyproxy/envoy/pull/37591/commits/d0a3b622e5727e52cee9117de5dba73b8a57419b.